scenario/ops_main_mock.py

```diff
@@ -195,7 +195,11 @@ def commit(self):
         if not self._has_emitted:
             raise RuntimeError("should .emit() before you .commit()")
 
+        # emit collect-status events
+        ops.charm._evaluate_status(self.charm)
```

Comment:
I was wondering what actually wired up the statuses added in `collect_(app|unit)_status` to the resulting status in state? But I guess it's all hidden in here.

Reply:
eventually the result is a call to `status-set`, and that was wired up already. So there were no changes necessary to `mocking.py`.

But this makes me think: should we add something to facilitate verifying all of the individual statuses that went into the collect-status event, or are we happy with verifying that the highest priority one is what we think it is? 
Food for thought.